apps/browser-tests/src/App.tsx

```diff
@@ -105,15 +110,25 @@ function Transactions({
           ],
         });
       } else if (provider === "magiceden") {
-        signPsbtResponse = await magiceden.signPsbt(psbt.toPSBT(), {
-          network,
-          inputsToSign: [
-            {
-              address: inputAddressInfo.address,
-              signingIndexes: [0],
-            },
-          ],
-        });
+        const meWallet = satsConnectWallets!.find(
```

Comment:
If this is undefined, shouldn't we just throw an error?

Reply:
Fixed!